.packit.yml

```diff
@@ -0,0 +1,21 @@
+specfile_path: .ci/packit/rpm/systemd.spec
+synced_files:
+  - .packit.yaml
+  - src: .ci/packit/rpm/systemd.spec
+    dest: systemd.spec
+upstream_package_name: systemd
+downstream_package_name: systemd
+# git describe returns in systemd's case 'v245-xxx' which breaks RPM version
+# detection (that expects 245-xxxx'. Let's tweak the version string accordingly
+current_version_command: ["bash", "-c", "git describe --tags | grep -Eo '[0-9]+.*$'"]
+actions:
+  post-upstream-clone:
+    - "git clone https://src.fedoraproject.org/rpms/systemd .ci/packit/rpm --depth=1"
+    - "sed -ri '/^Patch[0-9]+\\:.+\\.patch/d' .ci/packit/rpm/systemd.spec"
+
+jobs:
+- job: copr_build
+  trigger: pull_request
+  metadata:
+    targets:
+    - fedora-rawhide-x86_64
```

Comment:
We could build for other arches here. Why no "all" ?

Reply:
This was just PoC so far, if it's even a viable option, building on other archs would be definitely a way to go.

Reply:
From the currently available rawhide chroots:

```
fedora-rawhide-aarch64
fedora-rawhide-armhfp
fedora-rawhide-i386
fedora-rawhide-s390x
fedora-rawhide-x86_64
```

only `x86_64` and `aarch64` are usable - the rest is either irrelevant (`i386`) or emulated (`s390x` and `armhfp`), thus taking a *really* long time (~4 hours for the `s390x` build).

Reply:
i386 is definitely relevant, we still support 32 bit builds.

I agree that emulated architectures should be dropped.

Reply:
Ah, mea culpa, I wasn't even sure if it's going to compile on i386 Fedora given the warning (`Not-released Koji packages, [more info](https://fedoraproject.org/wiki/Changes/Noi686Repositories) why you probably don't need this one`). However, it builds without issues and it's as fast as the other native archs, neat!